examples/hotrod/services/frontend/web_assets/assets.go

```diff
@@ -0,0 +1,7 @@
+package webassets
```

Comment:
didn't find an easy way to remove `web_assets/` dir name from the files when embedding